### PR TITLE
CDK Public Roadmap process - initial RFC

### PR DESCRIPTION
**Pull request by evgenyka**
This is a request for comments about CDK Public roadmap process . See #450 for
additional details. 

[Rendered Version](https://github.com/aws/aws-cdk-rfcs/blob/CDK-roadmap-updates/text/450-cdk-public-roadmap.md)

APIs are signed off by @{BAR_RAISER}.

---

_By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache-2.0 license_

### CODE REVIEW

File: text/450-cdk-public-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+### Why are we doing this?
+
+Current public roadmap is not up to date, which provdies ambiguety for the community.
+It is hard to update, hard to review and almost impossible to digest.
+
+### Why should we _not_ do this?
+
+We do have current public roadmap, need to invest time and effort to keep it actual.
+
+### What is the technical solution (design) of this feature?
```
Comment (rix0rrr) [inline text/450-cdk-public-roadmap.md:69]: A lot of our work is reactive (high priority issues coming in, PRs coming in).

How are you seeing that fit in to a monthly planning? Or are there buckets for "planned" and "unplanned" work?

File: text/450-cdk-public-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Today, users have a problem when they would like to know the future plans of CDK team.
+Users need a way to get frequent updates and mid/long term visibility into product roadmap.
+
+## Working Backwards
+
+**CHANGELOG**
+`BREAKING CHANGE` - the proposed will replace the current process for AWS CDK Roadmap updates.
+
+**README**
```
[the remark this answers is not in the document]
Comment (evgenyka) [reply text/450-cdk-public-roadmap.md:15]: OK! Will revise it as a Blog Post, thank for the suggestion!

File: text/450-cdk-public-roadmap.md (hunk 1 above)
Comment (evgenyka) [reply text/450-cdk-public-roadmap.md:69]: I believe we should plan for that: there should be "protected investment" buget-wise for KTLO (Keep The Lights On) tasks like p0/p1 issues, PR reviews, etc. But those are not part of the roadmap. 
The roadmap should be focused on Innovation and Tech Debt.

Comment (rix0rrr) [reply text/450-cdk-public-roadmap.md:74]: Okay. Can you put that in the doc? :)

Comment (evgenyka) [reply text/450-cdk-public-roadmap.md:74]: Sure, it is already there - see the last version #README section.

**Pull request closed without merging**